src/screens/Favorites/styles.ts

```diff
@@ -1,15 +1,27 @@
+import {RFValue} from 'react-native-responsive-fontsize';
 import styled from 'styled-components/native';
+import Text from '~/components/Text';
 
-export const Container = styled.View`
+export const Container = styled.SafeAreaView`
```

Comment:
```suggestion
export const Container = styled(SafeAreaView)`
```

use the safe area view from the react-native-safe-area-context

Reply:
dont forget the import

Reply:
Resolved in commit 8be304bef5499c5ee9916c22687b4abda5363ca6